app/views/profiles/_edit.haml

```diff
@@ -9,10 +9,22 @@
   = render 'profiles/edit_public', :profile => profile, :aspect => aspect, :person => person
 
   %hr
+  :javascript
```

Comment:
Yes, it becomes:  
<input name="profile[is_public]" id="profile_is_public" value="true" type="checkbox">

Yes, that what I mean.